src/redux/configureStore.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
+import bookReducer from './books/books';
+import categorieReducer from './categories/categories';
+
+export default combineReducers({
+  bookReducer,
+  categorieReducer,
+});
```

Comment:
- [ ] You combined the reducers in a good way, but you still need to create a store by using `configureStore`. Try to import `configureStore` from `@reduxjs/toolkit` and use it to create a store. You may check out this link for more information: [link](https://redux-toolkit.js.org/api/configureStore)